card/spec/lib/card/reference_spec.rb

```diff
@@ -98,93 +108,98 @@
     watermelon_seeds = newcard('watermelon+seeds', 'black')
     lew = newcard('Lew', "likes [[watermelon]] and [[watermelon+seeds|seeds]]")
 
-    assert_equal [1,1,1,1], lew.references_to.map(&:present), "links should not be Wanted before"
+    assert_equal [1,1,1,1], lew.references_to.map(&:present),
+      "links should not be Wanted before"
+
     watermelon = Card['watermelon']
     watermelon.update_referencers = false
-    watermelon.name="grapefruit"
+    watermelon.name = 'grapefruit'
     watermelon.save!
-    expect(lew.reload.content).to eq("likes [[watermelon]] and [[watermelon+seeds|seeds]]")
-    assert_equal lew.references_to.order(:id).map(&:ref_type), ['L','P','P','L'], "links should be a LINK"
-    assert_equal lew.references_to.order(:id).map(&:present), [ 0, 0, 1, 0 ],  "only reference to +seeds should be present"
+    correct_content = 'likes [[watermelon]] and [[watermelon+seeds|seeds]]'
+    expect(lew.reload.content).to eq(correct_content)
+
+    ref_types = lew.references_to.order(:id).map(&:ref_type)
+    assert_equal ref_types, ['L','P','P','L'], "links should be a LINK"
```

Comment:
Space missing after comma.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.